content/en/docs/how-to/registry-authentication.md

```diff
@@ -17,7 +17,69 @@ To authenticate to an OCI-compliant registry, use the `notation login` command w
 notation login -u <username> -p <password> <registry>
 ```
 
-> **Note:** If `notation login` is failing, you may need to configure [Docker Credential Store](https://github.com/notaryproject/notation/blob/main/specs/registry-auth.md#credential-store) for your environment.
+> **Note:** If `notation login` is failing, you may need to configure [Docker Credential Store](https://github.com/notaryproject/notation/blob/main/specs/registry-auth.md#credential-store) for your environment or configure environment variables as detailed in the *Configure Docker Credential Store for Linux* section.
+
+## Configure environment variables to authenticate to an OCI-compliant registry
+
+Set the `NOTATION_USERNAME` and `NOTATION_PASSWORD` environment variables to authenticate to an OCI-compliant registry.
+
+```console
+export NOTATION_USERNAME="YOUR_REGISTRY_USERNAME"
+export NOTATION_PASSWORD="YOUR_REGISTRY_PASSWORD"
+```
+
+After the environment variables are set, you can use `notation login` without specifying a username or password.
```

Comment:
user doesn't need to use notation login after env variable.

Reply:
+1

Reply:
Updated